.env.sample

```diff
@@ -7,6 +7,9 @@ SND_SERVER_PORT=6660
 SND_SQLITE_DB_PATH=///./data.sqlite
 SND_RECOMMENDER_ARCHIVE_PATH=./recommender_model.zip
 
+SND_SECRET_KEY=<secret_key>
+ALGORITHM=<algorithm_name>
```

Comment:
Could use a bit more specific name perhaps? :) `SND_ENC_ALGORITHM`?
Also not sure if it even has to be configurable, do you see when we want different algs across different environments?